javascript/advanced_examples/gdx_start_stop_desmos.html

```diff
@@ -91,63 +86,75 @@
     updateTable();
    
     const selectDeviceBtn = document.querySelector('#select_device');
+    const collectBtn = document.querySelector('#start_stop_collection');
     const output = document.querySelector('#output'); 
     let count = 0;
     // create global object gdxDevice to be used in other 
     let gdxDevice = "";  
```

Comment:
just a note, I know this isn't from your PR but for future updates: we shouldn't pre-define an object as a string.